libcxx/test/std/utilities/text_encoding/text_encoding.members/text_encoding.aliases_view/begin.pass.cpp

```diff
@@ -0,0 +1,65 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <text_encoding>
+
+// REQUIRES: std-at-least-c++26
+
+// struct text_encoding::aliases_view
```

Comment:
```suggestion
// REQUIRES: std-at-least-c++26

// <text_encoding>

// struct text_encoding::aliases_view
```

We put the `REQUIRES`, `UNSUPPORTED`... etc. on top and then the synopsis: the header name, etc.